contracts/FIND.cdc

```diff
@@ -1182,10 +1199,21 @@ pub contract FIND {
 		pub case LOCKED
 	}
 
+	pub resource interface NetworkPublic{
```

Comment:
I think this is not necessary anymore. 
Because there will always be only 1 network (which is FIND) and the FIND contract and FindLeaseMarkets will be deployed to the same account.

Reply:
Ok, then I think we should remove it.